include/bluetooth/services/bas_c.h

```diff
@@ -192,6 +213,39 @@ int bt_gatt_bas_c_read(struct bt_gatt_bas_c *bas_c, bt_gatt_bas_c_read_cb func);
  */
 int bt_gatt_bas_c_get(struct bt_gatt_bas_c *bas_c);
 
+/**
+ * @brief Check if notification is supported by service.
+ *
+ * @param bas_c BAS Client object.
+ *
+ * @return true If notifications are supported.
+ *              Otherwise false is returned.
+ */
+static inline bool bt_gatt_bas_c_notify_supported(struct bt_gatt_bas_c *bas_c)
+{
+	return bas_c->notify;
+}
+
+/**
+ * @brief Read periodically the battery level value from the device with
+ *        specific time interval.
+ *
+ * This function sends a read request to the connected device periodically.
+ * Can be used only when BAS do not support notification. Time interval
+ * K_NO_WAIT or K_FOREVER causes reading charasteristic value as soon as possible.
+ * Interval can be changed during reading If reading procces is in progress then
+ * next reading is started with new interval.
+ *
+ * @param bas_c BAS Client object.
+ * @param interval Characteristic Read interval in miliseconds.
+ * @param func The callback function.
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int bt_gatt_bas_c_read_interval_set(struct bt_gatt_bas_c *bas_c, s32_t interval,
```

Comment:
`read_interval_set` would imply that this function only set the interval. But it does more than that.
how about `periodic_read_start`?

Reply:
Ok sounds better.